docker/build/tegra_cyber.aarch64.dockerfile

```diff
@@ -7,22 +7,29 @@ ARG INSTALL_MODE
 LABEL version="1.2"
 
 ENV DEBIAN_FRONTEND=noninteractive
-ENV PATH /opt/apollo/sysroot/bin:$PATH
-
-COPY installers /tmp/installers
-COPY rcfiles /opt/apollo/rcfiles
-# Pre-downloaded tarballs
-COPY archive /tmp/archive
-
-RUN bash /tmp/installers/install_minimal_environment.sh ${GEOLOC}
-RUN bash /tmp/installers/install_bazel.sh
-RUN bash /tmp/installers/install_cmake.sh ${INSTALL_MODE}
-RUN bash /tmp/installers/install_llvm_clang.sh
 
-RUN bash /tmp/installers/install_cyber_deps.sh
-RUN bash /tmp/installers/install_qa_tools.sh
-
-RUN bash /tmp/installers/install_visualizer_deps.sh
-RUN bash /tmp/installers/post_install.sh ${BUILD_STAGE}
+ENV PATH /opt/apollo/sysroot/bin:$PATH
 
-WORKDIR /apollo
+COPY installers/installer_base.sh /tmp/installers/
+COPY installers/install_nvidia_ml_for_jetson.sh /tmp/installers/
+COPY rcfiles/apollo.sh.sample /opt/apollo/rcfiles/
+
+RUN bash /tmp/installers/install_nvidia_ml_for_jetson.sh
+
```

Comment:
Used to create base image for `aarch64` : x86_64's  `nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04` equivalent.